src/nodes/NodeManager.ts

```diff
@@ -600,11 +601,13 @@ class NodeManager {
    * Retrieves all the vaults for a peers node
    */
   @ready(new nodesErrors.ErrorNodeManagerNotRunning())
-  public async scanNodeVaults(nodeId: string): Promise<Array<string>> {
+  public async scanNodeVaults(
```

Comment:
Not sure if we are still proxying all calls. But I guess this will be fixed in our node management refactoring #310